admin/views/partial-settings-tab-video.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * @package WPSEO\Admin\Views
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+
+if ( ! empty( $tab_video_url ) ) :
+
+	?>
+	<div class="wpseo-tab-video-container">
+		<div class="wpseo-tab-video-container__handle">
```

Comment:
Then you should also look at [aria-controls](http://rawgit.com/w3c/aria/master/aria/aria.html#aria-controls)